src/ngraph/pass/constant_folding_unary.cpp

```diff
@@ -94,7 +94,12 @@ shared_ptr<op::Constant> fold_constant_unary(shared_ptr<op::Constant> constant,
             runtime::reference::negate<T>(
                 constant->get_data_ptr<T>(), out_vec.data(), shape_size(out_shape));
         }
-        else if (is_type<op::Not>(unary))
+        else if (is_type<op::v0::Not>(unary))
+        {
+            runtime::reference::logical_not<T>(
+                constant->get_data_ptr<T>(), out_vec.data(), shape_size(out_shape));
+        }
+        else if (is_type<op::v1::LogicalNot>(unary))
```

Comment:
This list should be kept in alphabetical order.

Reply:
Fixed